crates/modelardb_server/tests/integration_test.rs

```diff
@@ -671,19 +671,32 @@ fn test_can_collect_metrics() {
     // Check that all metrics are present in the response.
     let metrics_array = modelardb_common::array!(metrics, 0, StringArray);
 
-    assert_eq!(metrics_array.value(0), "used_uncompressed_memory");
-    assert_eq!(metrics_array.value(1), "used_compressed_memory");
-    assert_eq!(metrics_array.value(2), "ingested_data_points");
-    assert_eq!(metrics_array.value(3), "used_disk_space");
+    assert_eq!(metrics_array.value(0), "used_multivariate_memory");
+    assert_eq!(metrics_array.value(1), "used_uncompressed_memory");
+    assert_eq!(metrics_array.value(2), "used_compressed_memory");
+    assert_eq!(metrics_array.value(3), "ingested_data_points");
+    assert_eq!(metrics_array.value(4), "used_disk_space");
 
     // Check that the metrics are populated when ingesting and flushing.
     let values_array = modelardb_common::array!(metrics, 2, ListArray);
 
+    // The multivariate_memory metric should be recorded when data is received and ingested.
```

Comment:
Maybe `The used_multivariate_memory metric should record` instead.

Reply:
Fixed.